src/Components/VendorView/VendorView.js

```diff
@@ -1,16 +1,73 @@
-import React from 'react'
+import React, {useState, useEffect} from 'react'
 import Status from '../Status/Status'
 import TruckDetails from '../TruckDetails/TruckDetails'
+import { MenuItem } from '../MenuItem/MenuItem'
+import { fetchAllMenus } from '../../utilities/apiCalls'
+import './VendorView.css'
 
 const VendorView = ({ vendor, currentVendor, toggleLive }) => {
-  // console.log(vendor, 'vendor from vendor view')
+  const [menu, setMenu] = useState([])
+  const truncateUrl = (url) => {
+    const maxTextLength = 25;
+    return url.length > maxTextLength ? url.substr(0, maxTextLength) + '...' : url;
+  };
+
+
+  const fetchData = async () => {
+    try {
+      const menuData = await fetchAllMenus()
+      let currentMenu = menuData.data.attributes.filter((item) => item.vendor === vendor.id)
+      setMenu(currentMenu)
+    } catch(error) {
+      console.log(error, "error")
+    }
+      }
+      useEffect(() => {
+        fetchData()
+      }, []) 
+  console.log(vendor, 'vendor from vendor view')
+  const suggestion = vendor.down_rating > vendor.up_rating ? 'Maybe you should try going Open now!' : ''
   return(
-    <div>
-      {/* <button onClick={() => toggleLive(vendor)}>{vendor.status === 'true' ? "Close Shop" : "Go Live"}</button> */}
-    
-      <Status toggle={toggleLive} vendor={vendor}/>
-      {/* <TruckDetails vendors={vendors} currentVendor={currentVendor}/> */}
+    <div className="vendor-view">
+      <div className='status-container'>
+        <Status toggle={toggleLive} vendor={vendor} />
+      </div>
+    <div className="vendor-view-cont">
+      <div className="vv-left">
+        <div className="statistics-container">
```

Comment:
statistics container is a nice touch

Reply:
I like this spelled out in a way that's more user-friendly, especially since we don't want vendors to be able to vote on their own truck. Great idea!